src/cmd/sanity_test/restore/onedrive.go

```diff
@@ -19,7 +19,7 @@ const (
 	owner = "owner"
 )
 
-type permissionInfo struct {
+type PermissionInfo struct {
```

Comment:
to help minimize the possibility of circular dependencies, can this move to `sanity_test/utils`?